packages/forms-web-app/src/views/register/organisation/full-name.njk

```diff
@@ -4,6 +4,13 @@
 {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
 
+{% block script %}
+	{% set initiateScriptsConfig = (initiateScriptsConfig.push({
```

Comment:
what will happen if the user doesn't have JS enabled?

Reply:
also perhaps rather than repeating x3, this block could be defined in a separate njk file and imported into each of the 3 page templates

Reply:
Seems ok without javascript, just doesn't run the script from what I can see in browser debug.  Seems to work ok, we have a general problem with back button but that is not to do with this change.

I've been trying to move the script out of the njk into it's own njk file for import.  It basically fails because the initiateScript is not available -
Error: Unable to call `initiateScriptsConfig["push"]`, which is undefined or falsey

Think it's to do with the order of imports of the templates and which script are available at run time.  I may leave this one.